### ISSUE TITLE
insert_as_last_child method ignores custom primary key column

### ISSUE THREAD

**Issue by Alexis2004** [evopix-archive/orm-mptt#15]
Patch for bugfix is attached:

```
Index: modules/mptt/classes/kohana/orm/mptt.php
===================================================================
--- modules/mptt/classes/kohana/orm/mptt.php    (revision )
+++ modules/mptt/classes/kohana/orm/mptt.php    (revision )
@@ -322,7 +322,7 @@
     */
    public function insert_as_last_child($target)
    {
-       $target = $this->parent_from($target, 'id');
+       $target = $this->parent_from($target);
        return $this->insert($target, $this->right_column, 0, 1);
    }
```

**Comment by evopix** [comment]
You should actually just replace `'id'` with `$this->parent_column`. That way it can be overridden in the model. Thanks!

**Comment by Alexis2004** [comment]
Why `$this->parent_column` not `$this->_primary_key`? (this method use primary key if null is passed)

**Comment by evopix** [comment]
You're right, it should be `$this->_primary_key`. Sorry!

**Comment by evopix** [comment]
Closed by ab94d2a7e495090d0d605f3e81c617562e6ebff8. `insert_as_last_child` method ignores custom primary key column
